transport/nio/src/main/java/org/kaazing/gateway/transport/nio/NioSystemProperty.java

```diff
@@ -51,7 +51,8 @@
     // TCP_IDLE_TIMEOUT will kill the session if nothing is written or read at nio level.
     // Note, the idle usage is using the mina netty idle timeout which may be set
     // by higher layers. Logic for this is in NioIdleFilter
-    TCP_IDLE_TIMEOUT("org.kaazing.gateway.server.transport.tcp.IDLE_TIMEOUT", Integer.toString(0));
+    TCP_IDLE_TIMEOUT("org.kaazing.gateway.server.transport.tcp.IDLE_TIMEOUT", Integer.toString(0)),
+    UDP_IDLE_TIMEOUT("org.kaazing.gateway.server.transport.udp.IDLE_TIMEOUT", "60");
```

Comment:
`NioSystemProperty` probably needs a convenience constructor with `String` and `int` parameters that both of these idle-timeout constants can use directly.